junit-jupiter-api/src/main/java/org/junit/jupiter/api/DynamicContainer.java

```diff
@@ -65,13 +66,31 @@ public static DynamicContainer dynamicContainer(String displayName, Iterable<? e
 	 * @see #dynamicContainer(String, Iterable)
 	 */
 	public static DynamicContainer dynamicContainer(String displayName, Stream<? extends DynamicNode> dynamicNodes) {
-		return new DynamicContainer(displayName, dynamicNodes);
+		return dynamicContainer(displayName, null, dynamicNodes);
+	}
+
+	/**
+	 * Factory for creating a new {@code DynamicContainer} for the supplied display
+	 * name, the test source and stream of dynamic nodes.
```

Comment:
"the test source and stream of dynamic nodes." --> "test source, and stream of dynamic nodes."